ui-components/src/locales/bn.json

```diff
@@ -1173,7 +1173,6 @@
   "listingFilters.waitlist": "প্রতীক্ষার তালিকা",
   "listingFilters.applyFilter": "ফিল্টার প্রয়োগ করুন",
   "listingFilters.noResults": "কোন ফলাফল নেই",
-  "listingFilters.noResultsSubtitle": "সেরা ফলাফলের জন্য ফিল্টার সম্পাদনা বা অপসারণ করুন।",
```

Comment:
Are there no changes needed for Spanish?

Reply:
@ludtkemorgan Turns out the Spanish translations were missing for that string 👀